Viper/Viper/View/CountryViewController.swift

```diff
@@ -0,0 +1,120 @@
+// CountryViewController.swift
+// Copyright © Natasha Ananas. All rights reserved.
+
+import UIKit
+
+/// Экран со странами
+final class CountryViewController: UIViewController, CountryViewControllerInputProtocol {
+    func didSelectRow(county: Country) {}
```

Comment:
```suggestion
    func didSelectRow(county: Country) {}
```
нужен ли этот метод?